docs/workers.md

```diff
@@ -238,19 +241,40 @@ expressions:
     ^/_matrix/client/(api/v1|r0|v3|unstable)/joined_rooms$
     ^/_matrix/client/(api/v1|r0|v3|unstable)/search$
 
+    # Encryption requests
+    ^/_matrix/client/(api/v1|r0|v3|unstable)/keys/claim
+    ^/_matrix/client/(api/v1|r0|v3|unstable)/room_keys
+
     # Registration/login requests
     ^/_matrix/client/(api/v1|r0|v3|unstable)/login$
     ^/_matrix/client/(r0|v3|unstable)/register$
     ^/_matrix/client/unstable/org.matrix.msc3231/register/org.matrix.msc3231.login.registration_token/validity$
 
-    # Event sending requests
+    # Event sending requests (for the event stream writer)
     ^/_matrix/client/(api/v1|r0|v3|unstable)/rooms/.*/redact
     ^/_matrix/client/(api/v1|r0|v3|unstable)/rooms/.*/send
     ^/_matrix/client/(api/v1|r0|v3|unstable)/rooms/.*/state/
     ^/_matrix/client/(api/v1|r0|v3|unstable)/rooms/.*/(join|invite|leave|ban|unban|kick)$
     ^/_matrix/client/(api/v1|r0|v3|unstable)/join/
     ^/_matrix/client/(api/v1|r0|v3|unstable)/profile/
 
+    # Typing requests (for the typing stream writer)
+    ^/_matrix/client/(api/v1|r0|v3|unstable)/rooms/.*/typing
+
+    # Device requests (for the to_device stream writer)
+    ^/_matrix/client/(api/v1|r0|v3|unstable)/sendToDevice/
+
+    # Account data requests (for the account_data stream writer)
+    ^/_matrix/client/(api/v1|r0|v3|unstable)/.*/tags
+    ^/_matrix/client/(api/v1|r0|v3|unstable)/.*/account_data
+
+    # Receipts requests (for the receipts stream writer)
+    ^/_matrix/client/(api/v1|r0|v3|unstable)/rooms/.*/receipt
+    ^/_matrix/client/(api/v1|r0|v3|unstable)/rooms/.*/read_markers
+
+    # Presence requests (for the presence stream writer)
+    ^/_matrix/client/(api/v1|r0|v3|unstable)/presence/.*/status$
```

Comment:
hrm, this feels a bit confusing, because we're talking about stream writers before we introduce them.

(also, there's something to be said for this being a static list of endpoints, that can be C&Ped wholesale).

Would it make more sense for this to move into the "Stream writers" section, with separate lists for each stream writer?

Reply:
hum, or maybe not, because even if we leave these things on the master, then the generic worker will proxy them to the right place?

in which case, maybe we should actually maintain two copies of this part of the list, one here under "generic worker" where we comment on the fact that `POST`s and `PUT`s will be proxied, and another *under* the "stream writers" section?

Reply:
I struggled with how to include them!

Having them twice is probably clearest, but there's concern about drift between them. I agree I don't like introducing this before the "stream writers" section though.

Reply:
The only reasonable thing I could think of was to list them twice, lease let me know what you think!